### PR TITLE
Assemble changes for 1.8.0 release

### PR DESCRIPTION
**Pull request by buberdds**
Depends on #1645

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,136 @@ The format is inspired by [Keep a Changelog].
...
+  ([#1641](https://github.com/oasisprotocol/oasis-wallet-web/issues/1641))
+
+### Bug Fixes and Improvements
+
+- Export private key re-design
+  ([#1704](https://github.com/oasisprotocol/oasis-wallet-web/issues/1704),
+  [#1720](https://github.com/oasisprotocol/oasis-wallet-web/issues/1720))
+- Adjust gas cost for Sapphire and Emerald
+  ([#1712](https://github.com/oasisprotocol/oasis-wallet-web/issues/1712))
+- Don't shuffle inactive validators to the top
```
Comment (tjanez) [inline CHANGELOG.md:57]: This one seems to be a duplicate (we already have this entry for 1.7.3 release).

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,136 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 1.8.0 (2023-10-18)
+
+### Process Changes
+
+- Add Change Log and the Change Log fragments process for assembling it
+  ([#1560](https://github.com/oasisprotocol/explorer/issues/1560))
```
[the remark this answers is not in the document]
Comment (buberdds) [reply CHANGELOG.md:20]: I will update towncrier.toml in https://github.com/oasisprotocol/oasis-wallet-web/pull/1645 and regen

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,134 @@ The format is inspired by [Keep a Changelog].
...
+
+- Miscellaneous improvements
+  ([#1614](https://github.com/oasisprotocol/oasis-wallet-web/issues/1614),
+  [#1623](https://github.com/oasisprotocol/oasis-wallet-web/issues/1623),
+  [#1624](https://github.com/oasisprotocol/oasis-wallet-web/issues/1624),
+  [#1621](https://github.com/oasisprotocol/oasis-wallet-web/issues/1621),
+  [#1637](https://github.com/oasisprotocol/oasis-wallet-web/issues/1637),
+  [#1639](https://github.com/oasisprotocol/oasis-wallet-web/issues/1639),
+  [#1640](https://github.com/oasisprotocol/oasis-wallet-web/issues/1640))
+- Dependency updates
```
Comment (tjanez) [inline CHANGELOG.md:74]: Something to consider for the future:

Should we introduce a separate section and Change Log fragment extension just for Dependency updates?

I think there are many more dependency updates here than in some average Go project.

**Review by tjanez — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (tjanez) [reply CHANGELOG.md:74]: Filled https://github.com/oasisprotocol/oasis-wallet-web/issues/1730

**Pull request merged**